lib/vagrant.pm

```diff
@@ -0,0 +1,35 @@
+package vagrant;
+use testapi;
+use strict;
+use warnings;
+use utils;
+
+our @ISA    = qw(Exporter);
+our @EXPORT = qw(setup_vagrant);
+
+# - install vagrant and all additional packages
+# - launch the required daemons
+#
+# Parameters:
+# - install_virtualbox: bool
+#   whether to install & setup virtualbox next to libvirt
+sub setup_vagrant {
+    my $install_virtualbox = shift;
+
+    select_console('root-console');
+
+    zypper_call('in vagrant vagrant-libvirt');
+
+    assert_script_run('systemctl start libvirtd');
+
+    assert_script_run('usermod -a -G libvirt bernhard');
+
+    if ($install_virtualbox) {
```

Comment:
Shouldn't we have an `if/else` to install either libvirt or virtualbox?

Reply:
Wouldn't that be inconvenient when you want libvirt **and** virtualbox?

Reply:
I think you test `libvirt` in one test and `virtualbox` on another test. So, it makes sense to install the only part needed for the current test.